src/uct/ib/dc/accel/dc_mlx5.c

```diff
@@ -51,7 +52,13 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_ep_t, uct_iface_t *tl_iface,
     }
 
     if (is_global) {
-        ucs_error("dc_mlx5 transport does not support global address");
+        char buf[64];
```

Comment:
minor: you declare buf here, but ah_attr (needed in this branch only) is declared with wider scope.